apps/windows/readme.md

```diff
@@ -0,0 +1,38 @@
+# Running the FluentUI Tester in UWP
+
+## Prerequisites
+
+- [Node.js](https://nodejs.org/en/download/)
+- [React Native Windows Development Dependencies](https://microsoft.github.io/react-native-windows/docs/rnw-dependencies)
+  - **NOTE:** Please make sure you grab all of the items listed there and the appropriate versions.
+- [Azure Credentials Plugin for NuGet](https://github.com/microsoft/artifacts-credprovider#manual-installation-on-windows)
+  - We're currently on an expermental version of react-native-windows which is being published to a private NPM feed hosted on Azure DevOps.
+
+### Add the App Signing Certificate to your Personal Certificate Store
+
+As with normal UWP app development, the UWP test harness that loads the JS bundle must have a developer-signed certificate present to install the APPX package.
+
+1. Windows+R to open the Run Dialog and type in `certmgr.msc`. **NOTE:** For some reason this takes some time on my machine.
+2. Expand `Personal` and right click on `Certificates`
+3. Under `All tasks` click `Import`
+4. For the Current User, `Browse` to find `FluentTester_Temporary.pfx` in your local clone of the repo. It is currently under [fluentui-react-native/apps/windows/windows/FluentTester](windows/FluentTester).
+5. On the next page, type in 'password' for the password.
+   ![Certificate Manager Browse Window](./../../assets/CertFile.png)
+
+## Running the App
+
+1. Run `yarn` from the root of the repo
+2. From this directory, simply `yarn windows`
```

Comment:
This is a bit confusing. It looks (to me), that 'yarn windows' should be run from the root of the repo, which isn't true. I would simply add the path apps/windows to #2 to avoid confusion.